.circleci/config.yml

```diff
@@ -23,7 +23,7 @@ executors:
 
   default-machine:
     machine:
-      image: ubuntu-1604:201903-01
+      image: ubuntu-2004:202201-02
```

Comment:
AFAIK, we were suggested to use this image: `ubuntu-2204:2023.04.2`
https://github.com/mojaloop/central-ledger/blob/main/.circleci/config.yml#L173